### PR TITLE
Rename homepageUrl

### PR DESCRIPTION
**Pull request by korycins**
- rename homepageUrl to homepageOnMarketplaceUrl

### CODE REVIEW

File: saleor/app/app_manifest_sample.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+    "version": "1.0.0",
+    "name": "My Wonderful App",
+    "about": "My Wonderful App is a wonderful App for Saleor.",
+    "permissions": ["MANAGE_USERS", "MANAGE_STAFF"],
+    "appUrl": "http://localhost:3000/app",
+    "configurationUrl": "htpp://localhost:3000/configuration",
+    "tokenTargetUrl": "http://localhost:3000/register",
+    "dataPrivacy": "Lorem ipsum",
+    "dataPrivacyUrl": "http://localhost:3000/app-data-privacy",
+    "homepageOnMarketUrl": "",
```
Comment (maarcingebala) [inline saleor/app/app_manifest_sample.json:12]: Why such a weird name?

Comment (korycins) [reply saleor/app/app_manifest_sample.json:12]: So what do you propose?

Comment (korycins) [reply saleor/app/app_manifest_sample.json:12]: This should be a URL which points on App's page on the marketplace

Comment (maarcingebala) [reply saleor/app/app_manifest_sample.json:12]: What's wrong with `homepageUrl`?

Comment (korycins) [reply saleor/app/app_manifest_sample.json:12]: It doesn't emphasize that we expect the marketplace's URL.  We need to have a URL which we will pin to button `Visist on marketplace`

**Pull request closed without merging**